processor/routingprocessor/router.go

```diff
@@ -394,7 +394,7 @@ func (r *router) registerTracesExporters(hostTracesExporters map[config.Componen
 // registerExportersForRoutes registers exporters according to the configuring
 // routing table, taking into account the provided map of available exporters.
 func (r *router) registerExportersForRoutes(available ExporterMap) error {
-	r.logger.Debug("Registering exporters for routes", zap.Any("exporters", available))
```

Comment:
Can we get a list of exporter names out of this map, instead of just removing it?

Reply:
We could but I'd need to iterate over the whole map and save it in a slice (conditionally on a log level? Can this be easily done?). Would that be ok?

Reply:
Not sure it's worth it -- it would certainly be helpful to know the available exporters and the requested for the route, but perhaps the extra code wouldn't be worth it...